packages/cactus-test-tooling/src/main/typescript/fabric/fabric-test-ledger-v1.ts

```diff
@@ -323,6 +370,815 @@ export class FabricTestLedgerV1 implements ITestLedger {
+    }
+
+    if (this.stateDatabase !== database) {
+      log.warn(
+        "Adding an organization with a different state database than org1 and org2",
+      );
+    }
+
+    const peerPortNumber = Number(peerPort);
+    const mspId = orgName + "MSP";
+
+    const ssh = new NodeSSH();
+    const sshConfig = await this.getSshConfig();
+    await ssh.connect(sshConfig);
+    log.debug(`SSH connection OK`);
+
+    try {
+      //Avoid well-known ports
+      if (peerPortNumber < 1024) {
+        throw new Error(`${fnTag} Invalid port, port too small`);
+      }
+      const couchDbPort = Math.abs(peerPortNumber - 1067).toString();
+      const caPort = (peerPortNumber + 3).toString();
+
+      temp.track();
+
+      //const tmpDirPrefix = `hyperledger-cactus-${this.className}`;
+      //const tmpDirPath = temp.mkdirSync(tmpDirPrefix);
+      /*
+      const tmpDirPrefix = `hyperledger-cactus-${this.className}`;
+      const tmpDirPath = temp.mkdirSync(tmpDirPrefix);
+      const tmpDirPathDocker = temp.mkdirSync(
+        path.join(tmpDirPrefix, "docker"),
+      );
+    */
+
+      //
+
+      const dockerOutputAddOrgXDirectoryPath = path.join(
+        addOrgXDirectoryPath,
+        "docker",
+      );
+      // Calculate files to pass over
+      const couchdbFile = await this.populateFile(
+        addOrgXDirectoryPath,
+        "couch",
+        orgName,
+        couchDbPort,
+        dockerOutputAddOrgXDirectoryPath,
+      );
+      const caFile = await this.populateFile(
+        addOrgXDirectoryPath,
+        "ca",
+        orgName,
+        caPort,
+        dockerOutputAddOrgXDirectoryPath,
+      );
+
+      const composeFile = await this.populateFile(
+        addOrgXDirectoryPath,
+        "compose",
+        orgName,
+        peerPort,
+        dockerOutputAddOrgXDirectoryPath,
+      );
+
+      const cryptoFile = await this.populateFile(
+        addOrgXDirectoryPath,
+        "crypto",
+        orgName,
+        peerPort,
+        addOrgXDirectoryPath,
+      );
+
+      const configTxGenFile = await this.populateFile(
+        addOrgXDirectoryPath,
+        "configTxGen",
+        orgName,
+        peerPort,
+        addOrgXDirectoryPath,
+      );
+
+      // Pass add-Org files
+
+      const sourceFiles = [
+        couchdbFile,
+        caFile,
+        composeFile,
+        cryptoFile,
+        configTxGenFile,
+      ];
+
+      // Code taken from Fabric's Connector
+      Checks.truthy(sourceFiles, `${fnTag}:sourceFiles`);
+      Checks.truthy(Array.isArray(sourceFiles), `${fnTag}:sourceFiles array`);
+
+      /*
+      for (const sourceFile of sourceFiles) {
+        const { filename, filepath, body } = sourceFile;
+        const relativePath = filepath || "./";
+        const subDirPath = path.join(tmpDirPath, relativePath);
+        fs.mkdirSync(subDirPath, { recursive: true });
+        const localFilePath = path.join(subDirPath, filename);
+        fs.writeFileSync(localFilePath, body, "base64");
+      }
+      */
```

Comment:
@RafaelAPB Recommending to remove the comments if they are just dead-ends (same considerations apply as above)

```suggestion
```